src/modules/auth/store/useAuthStore.ts

```diff
@@ -0,0 +1,13 @@
+import { create } from 'zustand';
```

Comment:
we dont use states with zustand, just react-query

Reply:
i'll change it